core/Cargo.toml

```diff
@@ -0,0 +1,76 @@
+[package]
+name = "subxt-core"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+publish = true
+
+license.workspace = true
+readme = "README.md"
+repository.workspace = true
+documentation.workspace = true
+homepage.workspace = true
+description = "Sign extrinsics to be submitted by Subxt"
+keywords = ["parity", "subxt", "extrinsic", "no-std"]
+
+[features]
+default = ["std"]                                                                                                                                                # "std"
+std = [
+    "scale-info/std", 
```

Comment:
why are not all crates that has `default-features = false` added here?

For instance codec/std is not enabled and many more....